ajax/libs/web3/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "web3",
+  "version": "0.20.2",
+  "description": "Ethereum JavaScript API",
+  "filename": "web3.min.js",
+  "license": "LGPL-3.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ethereum/web3.js.git"
+  },
+  "homepage": "https://github.com/ethereum/wiki/wiki/JavaScript-API",
```

Comment:
(Just discussion not request)
Not sure if https://web3js.readthedocs.io/en/1.0/index.html is a better choice